deployment/group_vars/dai.yml

```diff
@@ -1,24 +1,39 @@
 ## General settings
 BRIDGE_MODE: "ERC_TO_NATIVE"
+HOME_NATIVE_NAME: "POA"
```

Comment:
Is my understanding correct and this template is not only for testing in Molecule?
If so, 'xDai' should be here, "xDai chain" for 'HOME_NAME' and 'Ethereum Mainnet' for `FOREIGN_NAME`

Reply:
Correct, it can be used for real deployment and is used in Molecule for testing.
Updated the values as suggested.